client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/components/participant-detail-view/participant-detail-view.component.ts

```diff
@@ -342,6 +346,21 @@ export class ParticipantDetailViewComponent extends BaseMeetingComponent {
         }
     }
 
+    public enableSaveButton(): boolean {
+        return this.isFormValid && !this.lockedOutAndCanManage();
+    }
+
+    public lockedOutAndCanManage(): boolean {
+        const lockedOutHelper = this.personalInfoFormValue?.locked_out ?? this.user?.is_locked_out;
+        return lockedOutHelper && this.checkSelectedGroupsCanManage();
+    }
+
+    public disableLockoutCheckbox(): boolean {
+        const notChanged = (this.personalInfoFormValue?.locked_out ?? null) === null;
+        const isLockedOut = this.user?.is_locked_out;
+        return notChanged && !isLockedOut && this.checkSelectedGroupsCanManage();
+    }
+
```

Comment:
I would prefer computed properties. Also the method names sound like they would mutate something.

Reply:
Okay, I changed it and have used computed properties and renamed the properties.